### PR TITLE
Renaming container prop on Text component

### PR DESCRIPTION
**Pull request by Bidjii**

### CODE REVIEW

**Comment by Bidjii** [comment]
Changelog updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - **[FIX]** Handling of the colors on the `<PinIcon>`
+- **[UPDATE]** Change name of the `container` prop on the `<Text>` component into `tag` (harmonization)
```
Comment (pith) [inline CHANGELOG.md]: "breaking change" would be more appropriate

Comment (loick) [reply CHANGELOG.md]: Agreed

File: src/text/index.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ export enum TextDisplayType {
   TITLESTRONG = 'titleStrong',
 }
 
-export enum TextContainerType {
+export enum TextTagType {
```
Comment (loick) [inline src/text/index.tsx]: TagType might be enough

Comment (Bidjii) [reply src/text/index.tsx]: I'll keep it that way to be consistant with the `TextDisplayType` above

**Pull request merged**